public/pages/createDetector/components/DetectorInfo/DetectorInfo.tsx

```diff
@@ -25,23 +25,26 @@ interface DetectorInfoProps {
 }
 function DetectorInfo(props: DetectorInfoProps) {
   return (
-    <ContentPanel title="Define detector" titleSize="s">
-      <EuiFormRow>
-        <EuiText size="s">
-          To define a detector, you start by providing a name and description.
-        </EuiText>
-      </EuiFormRow>
+    <ContentPanel title="Name and description" titleSize="s">
       <Field name="detectorName" validate={props.onValidateDetectorName}>
         {({ field, form }: FieldProps) => (
           <EuiFormRow
-            label=" Detector name"
+            label={
+              <div>
+                <p>Name</p>
+                <p className="sublabel">
+                  Specify a unique and descriptive name that is easy to
+                  recognize.
+                </p>
+              </div>
+            }
```

Comment:
We will need to define some spacing and fonts looks different. Could you verify with Lovinda and show her this screenshot? I believe we need to smaller the text for hints.

Reply:
Another recommendation , as this is the pattern I would suggest to create new Component and use it here. 

A component could take two arguments , title and hint and consume it here.

Reply:
The screenshot looks different than what it really does.  I updated screenshots. Also, I created a new component as you said.